src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformTabularTests.cs

```diff
@@ -291,9 +292,10 @@ public async Task SQL_CdpTabular_GetTables3()
             await connectorTable.InitAsync(client, $"/apim/sql/{connectionId}", CancellationToken.None, logger);
             Assert.True(connectorTable.IsInitialized);
 
-#pragma warning disable CS0618 // Type or member is obsolete
+//#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
>//# [](http://example.com/codeflow?start=0&length=3)

remove?

Reply:
This will be needed once we complete [https://github.com/microsoft/Power-Fx/issues/2940](https://github.com/microsoft/Power-Fx/issues/2940)
So keeping it for future.